src/app/(post-login)/submission/components/ProgramMenu.tsx

```diff
@@ -47,7 +47,68 @@ const StatusMenuItem: FC<{ children: ReactNode }> = ({ children }) => {
 	);
 };
 
-const parseGQLResp = (data: SideMenuQuery) => {
+const ProgramMenu = ({ searchQuery }: { searchQuery: string }) => {
+	const params = useParams();
+	const pathname = usePathname();
```

Comment:
I'm not going to request changes for this but...

When we have reliance on param and pathname variables it would probably good to call this out in a TSDoc before the component declaration. It would be best if every component/function had such a documentation block, but especially when there is some usage of URL parameters it would be nice to call out at the top that what the component relies on. This would be a comment that says "Uses query param: `shortName` representing the currently selected program's shortName" and "Uses the URL path to identify which page from the menu is selected. Available pages are `/submission`, `/registration`, and `/clinical-submission`." Or something similar but formatted better and not mixing up options from the multiple components declared in this file.